### ISSUE TITLE
Group by taxon and type in compare search

### ISSUE THREAD

**Issue by kshefchek** [monarch-initiative/phenogrid#258]
Phenogrid has the functionality to compare a set of input IDs against a profile of phenotypes.  However, the results are displayed in the order given and are not grouped by taxon or type.  For example:

https://beta.monarchinitiative.org/analyze/phenotypes?input_items=HP%3A0002322&mode=compare&target_species=all&limit=100&gene_list=NCBIGene%3A3975%2CMGI%3A1856874%2C+MGI%3A5646293

How hard would it be to have phenogrid group these by type and taxon?  This would be useful for our patient pages and ortholog-phenotype tabs.

**Comment by harryhoch** [comment]
@kshefchek, you mean the targets, right? Right now, they are ordered in columns by strength of search. We might be able to add arbitrary sorting, if that information of type and taxon always come across in the owlsim results. Does it?

**Comment by jmcmurry** [comment]
Agreed this is important. It is related to some existing tickets on the "analyze phenotypes" tag but might this be a useful way of thinking about it?

Facets:   | Display type:      |Column sort/group options: 
---------------|-----------------------------|--------------------------------
- Taxon         |- Summary by gene         |- Sort by match score
- Type           |- All variants                     |- Sort by number of matched phenotypes
                     |- All genotypes, etc          |- Sort by number of unmatched phenotypes
                     |- All models available for purchase |- Group orthologs in contiguous columns

**Comment by kshefchek** [comment]
Yes, I simply want "Homo sapiens genes (2)" and the separator brackets, the taxon and type are available in the compare call.  There may be some instances where taxon is not available (for example patients/cases until the next release).  In this case just "Cases" would do.

**Comment by harryhoch** [comment]
ok, these are definitely things we could discuss. @yuanzhou, what do you think? Is there any obstacle to sorting the columns?

**Comment by yuanzhou** [comment]
@kshefchek just to make sure I understand your idea. Below is the screenshot of your linked example.

![capture](https://cloud.githubusercontent.com/assets/195873/19521805/11fe31ee-95e3-11e6-95f6-cef69a7c5b8c.PNG)

Are you suggesting to group just the target(genes or genotypes) x-items? In this example, LHX1 is a human gene, and the rest two are mouse genes. Then I'll have two group labels "Homo sapiens genes (1)" on top of the first gene, and "Mus musculus genes (2)" on top of the rest?

**Comment by kshefchek** [comment]
Yes exactly
